library/src/main/java/MapUtils.java

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Stack;
+
+/** Provides static functions for some operations on Maps */
```

Comment:
maybe remove word "static" here? otherwise you would have to update the comment if you had non-static methods